### PR TITLE
Include non-recording cpu usage queries into separate yaml

### PR DESCRIPTION
**Pull request by kusumachalasani**
## Description

Please describe the issue or feature and the summary of changes made to fix this. 
Include non-recording cpu usage queries into separate yaml and jsons to run the local monitoring APIs with any version of prometheus.  

Fixes # (issue)

### Type of change

- [ X] Bug fix
- [ ] New feature
- [ ] Docs update
- [ ] Breaking change (What changes might users need to make in their application due to this PR?)
- [ ] Requires DB changes

## How has this been tested?

Run the kruize-demos script by using the `resource_optimization_local_monitoring_norecordingrules` yaml.

- [ ] New Test X
- [ ] Functional testsuite

**Test Configuration**
- Testing is yet to complete.

## Checklist :dart:

- [ ] Followed coding guidelines
- [ ] Comments added
- [ ] Dependent changes merged
- [ ] Documentation updated
- [ ] Tests added or updated

## Additional information

Include any additional information such as links, test results, screenshots here

### CODE REVIEW

**Review by chandrams — COMMENTED** [review]
@kusumachalasani - Can you please update the local monitoring demo to use this file for minikube

**Comment by kusumachalasani** [comment]
Tested with the new jsons for minikube. CPU recommendations are observed. Working as expected.
image

**Comment by kusumachalasani** [comment]
> @kusumachalasani - Can you please update the local monitoring demo to use this file for minikube

Updated the demo script as part of PR https://github.com/kruize/kruize-demos/pull/90

**Review by chandrams — APPROVED** [review]
LGTM

File: manifests/autotune/performance-profiles/resource_optimization_local_monitoring_norecordingrules.json [hunk 1]
```diff
@@ -0,0 +1,395 @@
+{
+  "apiVersion": "recommender.com/v1",
+  "kind": "KruizePerformanceProfile",
+  "metadata": {
+    "name": "resource-optimization-local-monitoring"
```
Comment (dinogun) [inline manifests/autotune/performance-profiles/resource_optimization_local_monitoring_norecordingrules.json:5]: Do we want the name to be same as the other JSON?

Comment (kusumachalasani) [reply manifests/autotune/performance-profiles/resource_optimization_local_monitoring_norecordingrules.json:5]: As we are using the same performance profile name while creating the experiments for openshift and minikube in kruize-demos, I have kept the name same.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**